runtime/src/bank.rs

```diff
@@ -5843,6 +5843,35 @@ impl Bank {
         self.verify_transaction(tx, TransactionVerificationMode::FullVerification)
     }
 
+    /// Re-performs **feature** or state dependent verification on the transaction.
+    pub fn reverify_transaction(&self, tx: &impl SVMMessage) -> Result<()> {
+        // Loaded addresses depend on address table account state.
+        // We need only verify that loading works correctly.
+        // Since the address table is append only there is no chance addresses
+        // are different than the first time this was resolved.
+        let _loaded_addresses = self.load_addresses_from_ref(tx.message_address_table_lookups())?;
+
+        // Precompiles may be verified at this time if feature is not activated.
+        // Precompile verification can be dependent on features of the bank.
+        let move_precompile_verification_to_svm = self
+            .feature_set
+            .is_active(&feature_set::move_precompile_verification_to_svm::id());
+        if !move_precompile_verification_to_svm {
+            verify_precompiles(tx, &self.feature_set)?;
+        }
```

Comment:
As I was going through to write tests, I've become a bit unsure this is necessary.

Is there a way to **deactivate** a feature? iirc (at least on mnb) they are only activated.

If the transaction initially passed verification (will have to reach this point), then there's no way this could fail if we've only **activated** features since. If we can deactivate them, then this is necessary because a precompile's activation could have been removed

Reply:
Well in theory we could add a new precompile but yeah very unlikely to add one before we move precompile verification to the SVM. How about we skip precompile reverification and just add a comment to the `PRECOMPILES` static saying that if a new precompile is added, we will need to be mindful of reverifying them at epoch boundaries?

Reply:
Oh actually we should leave this. We verify the precompiles in the worker threads for parallelism. So this is not necessarily re-verifying, just verifying for first time.